app/views/pages/opportunities.html.erb

```diff
@@ -0,0 +1,3 @@
+<h1>Opportunities</h1>
+
+<p>Wowee! Lots of jobs here</p>
```

Comment:
Good placeholder page. I plan on replacing this with a scaffold down the road (so logged in Users can add opportunities)